samples/core/src/test/java/io/oasp/gastronomy/restaurant/general/common/RestTestClientBuilder.java

```diff
@@ -5,14 +5,16 @@
 
 import org.apache.cxf.common.util.Base64Utility;
 import org.apache.cxf.jaxrs.client.JAXRSClientFactoryBean;
+import org.springframework.stereotype.Component;
 
 import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider;
 
 /**
- * This class contains a static method to aid simulating a REST client.
+ * This class contains a method to aid simulating a REST client.
  *
  * @author jmolinar
  */
+@Component
```

Comment:
Please use JEE standards wherever possible (and avoid proprietary @Component, @Autowired, etc.)
https://github.com/oasp/oasp4j/wiki/guide-dependency-injection#example-bean